src/core/src/op/paged_attention.cpp

```diff
@@ -15,8 +15,72 @@ PagedAttentionExtension::PagedAttentionExtension(const ov::OutputVector& args) :
+                                                 const Output<Node>& subsequence_begins,
+                                                 const Output<Node>& block_indices,
+                                                 const Output<Node>& block_indices_begins,
+                                                 const Output<Node>& scale,
+                                                 const Output<Node>& sliding_window,
+                                                 const Output<Node>& alibi_slopes,
+                                                 const Output<Node>& max_context_len)
+    : Op({query,
+          key,
+          value,
+          key_cache,
+          value_cache,
+          past_lens,
+          subsequence_begins,
+          block_indices,
+          block_indices_begins,
+          scale,
+          sliding_window,
+          alibi_slopes,
+          max_context_len}) {
+    constructor_validate_and_infer_types();
+}
+
+PagedAttentionExtension::PagedAttentionExtension(const Output<Node>& query,
+                                                 const Output<Node>& key,
+                                                 const Output<Node>& value,
+                                                 const Output<Node>& key_cache,
+                                                 const Output<Node>& value_cache,
+                                                 const Output<Node>& past_lens,
+                                                 const Output<Node>& subsequence_begins,
+                                                 const Output<Node>& block_indices,
+                                                 const Output<Node>& block_indices_begins,
+                                                 const Output<Node>& scale,
+                                                 const Output<Node>& sliding_window,
+                                                 const Output<Node>& alibi_slopes,
+                                                 const Output<Node>& max_context_len,
+                                                 const Output<Node>& rotated_block_indices,
+                                                 const Output<Node>& rotation_deltas,
+                                                 const Output<Node>& rotation_trig_lut)
+    : Op({query,
+          key,
+          value,
+          key_cache,
+          value_cache,
+          past_lens,
+          subsequence_begins,
+          block_indices,
+          block_indices_begins,
+          scale,
+          sliding_window,
+          alibi_slopes,
+          max_context_len,
+          rotated_block_indices,
+          rotation_deltas,
+          rotation_trig_lut}) {
+    constructor_validate_and_infer_types();
+}
+
 void PagedAttentionExtension::validate_and_infer_types() {
-    OV_OP_SCOPE(PagedAttentionExtension_validate_and_infer_types);
+    OV_OP_SCOPE(_PagedAttentionExtension_validate_and_infer_types);
```

Comment:
```suggestion
    OV_OP_SCOPE(PagedAttentionExtension_validate_and_infer_types);
```
Probably leftover from deleting v16 opset